### DISCUSSION TITLE
漫画双页切分问题

### DISCUSSION THREAD

**Discussion by oubeichen** [xiaoyaocz/flutter_dmzj#102] [General]
以前的版本，在漫画阅读页有一个“使用网页API”的选项，开启能阅读部分无法阅读的也没，关闭之后会用移动版api自动双页切分，现在没有了。
现在应该是默认开启的，导致了双页漫画没法自动切分，合并成一页，在移动设备上阅读比较困难。

另外描述是支持MacOS的，貌似没见到Mac版本安装包？

**Comment by xiaoyaocz** [comment]
现在默认会先用移动版API（切页）请求，当移动版无法访问时会自动使用网页API（未切页）。
MacOS还有些问题，如需体验可以自行打包。

**Reply by oubeichen** [reply]
这就挺奇怪的，比如《C.M.B.森罗博物馆之事件目录》这部漫画。（直接搜C.M.B搜不到，搜森罗博物馆可以搜到，可能又是另一个问题）
打开任意一卷，旧版本切换移动版或者网页API 均可使用，并且移动版API能自动切页。
新版本打开只会显示未切页的样子。
同样的网络和设备。

**Reply by xiaoyaocz** [reply]
知道啥问题了，移动端API有两种清晰度的，现在默认会用高清的，而高清的是不切页的。
增加一个选项可以选择是否使用高清图 #105
